Documentation/network/clustermesh/aks-clustermesh-prep.rst

```diff
@@ -67,6 +67,16 @@ Install cluster one
     except for Azure reserved ranges and ranges used by other clusters we intend to
     add to the clustermesh.
 
+    .. note::
+        We need to set explicit values for pod CIDR and service CIDR, otherwise AKS will
+        allocate ``10.244.0.0/16`` for pod CIDR and ``10.0.0.0/16`` for service CIDR.
+        These are default values that are assigned by AKS while creating a cluster if 
+        the user doesn't specify pod CIDR and service CIDR arguments while creating 
+        the AKS clusters.
```

Comment:
I think the last `while creating the AKS clusters` here is redundant, can you remove it?

```suggestion
        the user doesn't specify pod CIDR and service CIDR arguments.
```